src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -37,6 +37,8 @@
     const DISABLE_TYPE_ENFORCEMENT = 'disable_type_enforcement';
     const SKIP_NULL_VALUES = 'skip_null_values';
 
+    const ARRAY_OBJECTS_KEY = 'array_objects';
```

Comment:
same here

Reply:
Ah, not used to all these fancy, not so new, php features yet